myokit/_sim/cvodessim.py

```diff
@@ -951,66 +974,45 @@ def set_min_step_size(self, dtmin=None):
 
     def set_fixed_form_protocol(self, times=None, values=None):
```

Comment:
I suppose we'd eventually want to move to some structure where we have a `Protocol` (?) base class, with subclasses `StepProtocol` and `TimeSeriesProtocol` (or similar) ?
Probably a good idea to delay that until after this PR?

Reply:
yea, I'd agree with that (and the delaying ;) )